src/views/HomeView.vue

```diff
@@ -104,13 +135,15 @@ export default {
   },
   data() {
     return {
+      tab: 'play',
       showSnackBar: false,
       snackBarMessage: '',
     };
   },
   computed: {
     ...mapState({
-      gameList: ({ gameList }) => gameList.games,
+      playableGameList: ({ gameList }) => gameList.openGames,
```

Comment:
Do you need a default here? 
```suggestion
      playableGameList: ({ gameList }) => gameList.openGames || [],
```
Ideally this should be handled in the whatever logic is happening in the store though.

Reply:
Yeah the store itself should always have a value for this which inits to an empty array, so I think this should be fine

Reply:
Nice, thanks.